pkgs/by-name/tm/tmuxai/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  versionCheckHook,
+  writableTmpDirAsHomeHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "tmuxai";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "alvinunreal";
+    repo = "tmuxai";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-L3PFyNkvEHfZ2pZFaurFKwHjhdZ+YBXtN2QuKFvktP4=";
+  };
+
+  vendorHash = "sha256-Ev3COZBVCACzCDbdKddJE67qMMzClZB9NHGNHHHaK7E=";
+
+  ldflags = [
+    "-s"
+    "-X=github.com/alvinunreal/tmuxai/internal.Version=v${finalAttrs.version}"
+    "-X=github.com/alvinunreal/tmuxai/internal.Commit=${finalAttrs.src.rev}"
+    "-X=github.com/alvinunreal/tmuxai/internal.Date=1970-01-01T00:00:00Z"
+  ];
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+    writableTmpDirAsHomeHook
+  ];
+
+  versionCheckKeepEnvironment = [ "HOME" ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  __darwinAllowLocalNetworking = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "AI-Powered, Non-Intrusive Terminal Assistant";
```

```suggestion
    description = "AI-Powered, Non-Intrusive Terminal Assistant";
    homepage = "https://github.com/alvinunreal/tmuxai";
```
Final request. Other than that, LGTM.

---

oops dumb of me, i thought from your prev comment you wanted me to _replace_ homepage with changelog, not just add changelog

wasn't thinking too hard i guess 😬

---

Haha np :)